server/app/core/stt_service.py

```
@@ -0,0 +1,48 @@
+# Copyright (c) 0235 Inc.
+# This file is licensed under the karakuri_agent Personal Use & No Warranty License.
+# Please see the LICENSE file in the project root.
+from faster_whisper import WhisperModel
+from app.schemas.agent import AgentConfig
+import numpy as np
```

Ruff (0.8.0)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)